packages/epetra/src/Epetra_Util.h

```diff
@@ -348,7 +348,7 @@ int Epetra_Util_insert_empty_positions(T*& array, int& usedLength,
     return(-1);
   }
 
-  if ((usedLength+numPositions) < allocatedLength) {
+  if (((unsigned int)usedLength+(unsigned int)numPositions) < (unsigned int)allocatedLength) {
```

Comment:
I can guess why compilers warned on this, but I'm not a huge fan of the change.  It turns undefined behavior on overflow, into well defined but totally wrong behavior on overflow.  I'm not sure what change would be better, though.

Reply:
I'm very open to alternatives.